src/lib/utils/SynapseClient.ts

```diff
@@ -3626,3 +3622,24 @@ export const forumSearch = (
     BackendDestinationEnum.REPO_ENDPOINT,
   )
 }
+
+/**
+ * Search through the history of access approvals filtering by accessor/submitter
+ * and optional by access requirement id. The caller must be a member of the ACT.
+ * https://rest-docs.synapse.org/rest/POST/accessApproval/search.html
+ * @param accessApprovalSearchRequest
+ * @param accessToken
+ */
+
+export const accessApprovalSearch = (
```

Comment:
Small nit, prefer name for functions to be a verb over a noun
```suggestion
export const searchAccessApprovals = (
```